backend/prometheus_app/core_services_monitoring.py

```diff
@@ -0,0 +1,71 @@
+from fastapi import FastAPI, Response
+from prometheus_client import Gauge, CollectorRegistry, generate_latest, CONTENT_TYPE_LATEST
+import docker
+import yaml
+
+app = FastAPI()
+client = docker.from_env()
+
+with open("config/monitored_services.yml", "r") as f:
+    config = yaml.safe_load(f)
+
+services_map = {entry["name"]: entry["container"] for entry in config["core_services"]}
+
+registries = {}
+gauges = {}
+
+for name, container_name in services_map.items():
+    reg = CollectorRegistry()
+    g = Gauge("oai_container_status", f"{container_name} container status", ["container", "status"], registry=reg)
```

Comment:
is this ok "oai_container_status" ? should be this unique ? do this make sens to have multiple gauge with same name ?